tests/test_fe_utils.py

```diff
@@ -103,3 +104,22 @@ def test_get_mol_name():
 
     mol.SetProp("_Name", "test_name")
     assert utils.get_mol_name(mol) == "test_name"
+
+
+def test_set_mol_coords():
+    mol = Chem.AddHs(Chem.MolFromSmiles("c1ccccc1"))
+    AllChem.EmbedMolecule(mol)
+
+    x0 = utils.get_romol_conf(mol)
+
+    # Make some random move
+    x1 = x0 + np.random.randn(*x0.shape)
```

Comment:
Should we set a seed above?

Reply:
Done in [4d77204](https://github.com/proteneer/timemachine/pull/813/commits/4d77204bd5168b5dfb06c163774094b2a851139a)